src/Microsoft.Diagnostics.Runtime/src/Utilities/Platform/LinuxFunctions.cs

```diff
@@ -96,6 +96,9 @@ internal override unsafe bool GetFileVersion(string dll, out int major, out int
             long address = dataOffset;
             long endAddress = address + dataSize;
 
+            Span<byte> bytes = stackalloc byte[1];
+            Span<char> chars = stackalloc char[1];
```

Comment:
I'm curious about this change, what is it for?  I don't care if this is just a cosmetic thing you added, but if this is due to a new build warning I'd like understand what it's warning about.

Reply:
Just like in C++, the memory allocated on the stack isn't freed after each iteration.

Reply:
Just like in C++, stack space allocated is only released at the end of the method.

Reply:
This is not cosmetic. By using `stackalloc` in a loop, the stack pointer is incremented each time the loop iterates. So we allocate N bytes on the stack instead of one, where only one is needed.